src/blas/KokkosBlas_gesv.hpp

```diff
@@ -93,6 +93,14 @@ gesv (const char pivot[],
       "Valid values include 'N' or 'n' (No pivoting), 'Y' or 'y' (partial pivoting).";
     Kokkos::Impl::throw_runtime_exception (os.str ());
   }
+#ifdef KOKKOSKERNELS_ENABLE_TPL_BLAS
+  if(!((pivot[0] == 'Y') || (pivot[0] == 'y'))) {
+    std::ostringstream os;
+    os << "KokkosBlas::gesv: pivot[0] = '" << pivot[0] << "'. " <<
+      "TPL BLAS does not support no pivoting.";
```

Comment:
Why not just dispatch to your hand-rolled implementation for this case?

Reply:
Thanks @mhoemmen. I totally agree with you. But, I think we do not have the implementation yet (I planned to do it in another PR). I think we only have the batched versions which are good for small size problems. @kyungjoo-kim and @srajama1, please correct me if I am wrong.